polyply/src/topology.py

```diff
@@ -211,20 +214,87 @@ class Topology(System):
         A dictionary of all typed parameter
     defines: list
         A list of everything that is defined
+    defaults: dict
+        A dict with default entries of GMX topfile
```

Comment:
Too vague to be actually useful. What keys? What do they mean?

Reply:
this is actually quite precise. The GROMACS top directive literally is called defaults, so it doesn't mean default entries but rather the entries in the default directive. I will rephrase though.